farm/data_handler/processor.py

```diff
@@ -383,7 +384,6 @@ def __init__(
         header=0,
         proxies=None,
         max_samples=None,
-        **kwargs
```

Comment:
This change is right IMO. But it breaks this test: https://github.com/deepset-ai/FARM/blob/5e74f8ad881a08c35bd23ee7561ed20271b2cf0c/test/test_processor_saving_loading.py#L18

The test should be fixed.

Reply:
Good point, I adjusted the test.

I am happy about the kwargs removal. This test is pretty outdated and should not have been working.